### PR TITLE
Announcement blog post

### PR DESCRIPTION
**Pull request by brandonjbjelland**

### CODE REVIEW

File: blog/2023-07-27-introducing-substratus.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+**Rapid experimentation with the latest ML models**: Substratus makes it easy
+for developers to fine-tune, serve, and experiment with open-source ML models in
+notebooks. This can be done in minutes not hours using our off-the-shelf
+loaders, trainers, and servers. All this is possible without writing any code.
+
+**Best practices baked-into your ML artifacts**: While Substratus is
+less-opinionated than other platforms about your team's exact workflow, we
+**are** rather opinionated about keeping core parts of the development process
+on rails. Once created, Models, Datasets, and Servers are all immutable. They
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:63]: This is not 100% true. I should reword.

File: blog/2023-07-27-introducing-substratus.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+auto-quanitization to enabling efficient serving whenever possible.
+
+**Open-core**: Substratus will always be open-core. Period. If your team can
+support it - and we believe many can - deploy the platform in as many places as
+you need and at whatever scale suits the business.
+
+While there are other open-source projects in this space, we've taken careful
+notes on how we want to differentiate.
+
+## What are our opinions as we launch?
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:76]: Are there any other design north stars we should highlight? To me, this answers the question of how we differ from the likes of kubeflow

File: blog/2023-07-27-introducing-substratus.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+1. **A small and consistent API**: We have custom resource definitions for
+   `Models`, `Datasets`, `Servers`, and `Notebooks`. That's it! The `.spec`s of
+   each resource share considerable commonality so there's little to keep in
+   your head.
+2. **Limited integration points with our supported cloud providers**: We are as
+   incentivized as you to avoid unnecessary cloud spend. To this end, we use
+   strictly commodity services with open-source on-prem equivalents or analogs
+   (e.g., S3, GCS, ABS, ceph, minio, etc.).
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:85]: Probably remove ceph here

File: blog/2023-07-27-introducing-substratus.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+3. **Large-scale distributed training**: We aim to support the largest of
+  training workloads. Currently we're able to utilize 8 GPUs for a single model
+  but we've heard this isn't enough for every use-case. We intend to make wider
+  horizontal GPU distribution a paved path on Substratus.
+4. **Partner Integrations**: We're teaming up with our friends at LangChain and
+   Weaviate to provide integrations and demos showcasing how and where
+   Substratus fits in the larger ML ecosystem.
+
+The roadmap above is all high-priority for our open-source project but we think
+there's great value in getting started at low or no-cost without, and without
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:115]: Stutter to correct here

File: blog/2023-07-27-introducing-substratus.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+   fully-aware of everywhere your org has deployed Substratus.
+4. **Cost and Capacity Conscious Workload Scheduling**: Cloud GPUs are
+   notoriously expensive and sometimes unavailable in certain locations.
+   Substratus can soften the blow by efficiently managing workloads with
+   cost-aware and capacity-conscious scheduling.
+
+What's next? Say hi on [discord](https://discord.gg/RcUShexGu8), file an [issue
+on GitHub](https://github.com/substratusai/substratus/issues/new) or [stack
+overflow](https://stackoverflow.com/questions/tagged/substratus) if you hit a
+snag, and sign-up for our monthly newsletter to keep track of the project. We're
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:136]: I think we should have this on our site before publishing

File: blog/2023-07-27-introducing-substratus.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+  clusters easily without needing to switch kubecontexts. Our enterprise CLI is
+   fully-aware of everywhere your org has deployed Substratus.
+4. **Cost and Capacity Conscious Workload Scheduling**: Cloud GPUs are
+   notoriously expensive and sometimes unavailable in certain locations.
+   Substratus can soften the blow by efficiently managing workloads with
+   cost-aware and capacity-conscious scheduling.
+
+What's next? Say hi on [discord](https://discord.gg/RcUShexGu8), file an [issue
+on GitHub](https://github.com/substratusai/substratus/issues/new) or [stack
+overflow](https://stackoverflow.com/questions/tagged/substratus) if you hit a
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:135]: Should we get rid of SO? Is it a dodo 🦤 now?

File: blog/2023-07-27-introducing-substratus.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+foundation of our fledgling SaaS product:
+
+1. **Substratus Cloud**: Soon, you'll be able get an API key and unlock the same
+   Substratus open-source API through a managed platform! Forget the quotas of
+   your personal cloud account, our SaaS is the perfect way to get started and
+   share your work without deploying infrastructure.
+2. **RBAC for Teams**: Role-based access control for teams, ensuring security
+   and privacy for your projects.
+3. **Multi-cloud, multi-cluster operations**: Manage your hybrid-SaaS Substratus
+  clusters easily without needing to switch kubecontexts. Our enterprise CLI is
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:126]: Is this how y'all envision the split in functionality being exposed? Maybe it's not important to get right here but I don't think we'll ship a separate enterprise cli

Comment (samos123) [reply blog/2023-07-27-introducing-substratus.md:126]: I would probably remove this part about enterprise CLI

File: blog/2023-07-27-introducing-substratus.md (hunk 6 above)
Comment (samos123) [reply blog/2023-07-27-introducing-substratus.md:135]: I think it's fine to keep it

File: blog/2023-07-27-introducing-substratus.md (hunk 5 above)
Comment (samos123) [reply blog/2023-07-27-introducing-substratus.md:136]: I would prefer it removed and publish this first before we dive into newsletters

File: blog/2023-07-27-introducing-substratus.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+  name: Brandon Bjelland
+  title: Co-founding Engineer, CXO
+  url: https://github.com/brandonjbjelland
+  image_url: https://avatars.githubusercontent.com/u/2502520?v=4
+tags: [hello world, introduction, oss launch]
+---
+# Announcing Substratus: A Multi-cloud Substrate for AI
+
+We're beyond excited to announce Substratus (s8s), an open-source platform to
+manage machine learning primitives in Kubernetes.
```
Comment (samos123) [inline blog/2023-07-27-introducing-substratus.md:14]: I'm not a huge fan of the s8s naming. Let Nick be the judge on this one. If it's 2v1 then I'm fine with it.

Comment (samos123) [reply blog/2023-07-27-introducing-substratus.md:14]: > We're beyond excited to announce Substratus (s8s), an open-source platform to
manage machine learning primitives in Kubernetes.

What you think about this instead?
"We're beyond excited to announce Substratus, an open-source platform to
Build, Train, and Serve state of the art Models on top of Kuberentes."

File: blog/2023-07-27-introducing-substratus.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+* Can I stream or do I really need to download this whole dataset?! I didn't
+  plan on uninstalling Flight Simulator tonight...✈️
+* Is any part of this workflow portable or repeatable? ♻️
+
+It would be reasonable to walk away from a night of yak-shaving with a Colab
+subscription and a handful of modest wins. But your plan to deliver a secure,
+consistent, and portable ML workflow to your day job is still effectively
+🤷🏾‍♀️. Even through great effort and ample research, your ultimate strategy
+might be: **hope**, that your engineering team is satisfied with Sagemaker's
+technical choices and your CFO doesn't care about pricing. 💸
```
Comment (samos123) [inline blog/2023-07-27-introducing-substratus.md:33]: I think this paragraph is slightly too long. I do like the points made, I'm just afraid the reader might never get to the Why Substratus part.

File: blog/2023-07-27-introducing-substratus.md (hunk 1 above)
Comment (samos123) [reply blog/2023-07-27-introducing-substratus.md:63]: once created the models are immutable from a substratus pov unless you directly go to GCS and start messing around

File: blog/2023-07-27-introducing-substratus.md (hunk 9 above)
Comment (brandonjbjelland) [reply blog/2023-07-27-introducing-substratus.md:33]: Totally fair. The scenario runs a bit long. I'll aim to chop by at least 1/3

File: blog/2023-07-27-introducing-substratus.md (hunk 8 above)
Comment (brandonjbjelland) [reply blog/2023-07-27-introducing-substratus.md:14]: I'm mostly just experimenting with s8s to see if it's sticky or not. Not opposed to removing it but I think esp early on we should experiment to see what if anything helps make the project easy to remember.

Comment (brandonjbjelland) [reply blog/2023-07-27-introducing-substratus.md:14]: I like your suggested replacement - I think management of ML primitives is mentioned elsewhere anyway.

Comment (samos123) [reply blog/2023-07-27-introducing-substratus.md:14]: Hmm that's a good point, s8s might stick better than substratus. Lets keep it for now.

File: blog/2023-07-27-introducing-substratus.md [hunk 10]
```diff
@@ -36,6 +36,11 @@ There's got to be a better way.
 
 ## Why Substratus?
 
+**Faster time to market of custom ML models**: Load a base model, finetune it
+with your own dataset and serve it without having to write any code. Substratus
+provides out of the box container images to make zero code finetuning a reality.
+Need more flexibility? Simply clone the official image and adapt it to your needs.
+
```
Comment (brandonjbjelland) [inline blog/2023-07-27-introducing-substratus.md:43]: This is a good point of emphasis. A note to self: re-org this list to put what's most compelling at the very top. Avoiding vendor lock-in is decidedly not our most important selling point.

**Review by nstogner — CHANGES_REQUESTED** [review]

**Review by samos123 — APPROVED** [review]

**Comment by brandonjbjelland** [comment]
Thank you for ushering this through! Beat me to it

**Pull request merged**